src/main/resources/synthetic.xml

```diff
@@ -11,7 +11,6 @@
     <type type="xldeploy.Task" extends="xlrelease.PythonScript" virtual="true">
         <property name="iconLocation" default="xldeploy/xldeploy-icon-black-small.png" hidden="true" />
         <property name="taskColor" hidden="true" default="#68B749" />
-
```

Comment:
Is there a reason why all these newlines are removed? Shouldn't they stay in the `xml` for readability?